### PR TITLE
Feature#22/create the user navbar

### PR DESCRIPTION
**Pull request by Javi1499**
# Description
What did you implemented/Why did you implemented this:
 - I created partial of user slideba following figma design
 - Bootstrap icon implemented 
 - Responsive design added
Why did you implemented this:
 - These changes will be use aftert to add the navigation using this component
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes
Desktop view
imagen

### CODE REVIEW

File: app/assets/stylesheets/utils/_variables.scss [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // Colors
-
+$red892:#892E2E;
 // Font Family
 
 // Text sizes
+// ViewPorts
```
Comment (YilKanda) [inline app/assets/stylesheets/utils/_variables.scss:6]: If you don't occupy these comments, delete them

File: app/assets/stylesheets/utils/_variables.scss [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+$mobile: 700px;
```
Comment (YilKanda) [inline app/assets/stylesheets/utils/_variables.scss:7]: Add a blank line

File: app/views/home/_user_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <li class="nav_section__item">
+      <i class="nav_section__icon bi bi-bag-dash-fill"></i>
+      <p class="nav_section__text">Ventas</p>
+    </li>
+    <li class="nav_section__item">
+      <i class="nav_section__icon bi bi-chat-fill"></i>
+      <p class="nav_section__text">Mensajes</p>
+    </li>
+  </ul>
+</div>
```
Comment (YilKanda) [inline app/views/home/_user_navbar.html.erb:31]: Add a blank line

**Review by EliasUR — APPROVED** [review]
LGTM

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by germanruzca — APPROVED** [review]
LGTM

**Pull request merged**